src/utils/es_utils.py

```diff
@@ -1,8 +1,10 @@
 """Elasticsearch API client utilities."""
 import json
 import requests
+import hashlib
 
 from src.utils.config import config
+import src.index_runner.es_indexer as es_indexer
```

Comment:
This introduces a circular dependency between es_utils and es_indexer.  That seems like a bad idea.  We should figure out what belongs where.